src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -789,14 +826,23 @@ public async Task PostAsync_ReuseRequestContent_Success(Configuration.Http.Remot
             }
         }
 
-        [OuterLoop("Uses external servers")]
+        [OuterLoop("Uses external servers", TestPlatforms.Any & ~TestPlatforms.Browser)]
         [Theory, MemberData(nameof(HttpMethodsThatDontAllowContent))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/53591", TestPlatforms.Browser)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/53592", TestPlatforms.Browser)]
         public async Task SendAsync_SendRequestUsingNoBodyMethodToEchoServerWithContent_NoBodySent(
             string method,
             Uri serverUri)
         {
+            if (method == "TRACE" && PlatformDetection.IsBrowser)
```

Comment:
Why modify the ActiveIssues to be of this form? If the problem occurs when method is `TRACE` I think the issue can reflect that instead. The ActiveIssue attribute should skip this method from being ran, and that would reflect in the test results. I'm not sure, but would the early return indicate this test method as passing (which sounds like it shouldn't be)?

Reply:
Same for the issue where method is `HEAD`

Reply:
Because this is data driven test, there are many combinations. Multiple endpoints/protocols x multiple HTTP verbs.
Most of them work well and we wouldn't like to disable them too. 

Yes, this reports false positive. This is how we deal with complex conditions elsewhere too. https://github.com/dotnet/runtime/blob/1e1951825bb58417b0566e4b641544483c9e8865/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs#L186